de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java

```diff
@@ -150,7 +150,7 @@ public void onSuccess(Counts result) {
     @Inject DEClientConstants deClientConstants;
     @Inject DEProperties deProperties;
     @Inject Provider<ErrorHandler> errorHandlerProvider;
-    @Inject Provider<DEFeedbackServiceFacade> feedbackServiceProvider;
+    @Inject Provider<DEUserSupportServiceFacade> userSupportServiceProvider;
```

Comment:
Another code-split potential -- if this is `AsyncProviderWrapper` instead of `Provider`, and the `.get()` below becomes a: 

```java
userSupportServiceProvider.get(new AsyncCallback<DEUserSupportServiceFacade>(){
    @Override
    public void onFailure (Throwable caught) {} // no-op, AsyncProviderWrapper doesn't care

    @Override
    public void onSuccess(DEUserSupportServiceFacade supportService) {
        supportService.submitSupportRequest(...)
    }
});
```

... then all the code of the service facade doesn't need to be there on initial load. This is used one other place, if I'm not mistaken, so that would need the same treatment or only the injection code would actually end up in the split.